packages/sdk/typescript/human-protocol-sdk/test/escrow.test.ts

```diff
@@ -0,0 +1,1368 @@
+
+      await expect(escrowClient.cancel(escrowAddress)).rejects.toThrow();
+
+      expect(escrowClient.escrowContract.cancel).toHaveBeenCalledWith();
+    });
+  });
+
+  describe('abort', () => {
+    test('should throw an error if escrowAddress is an invalid address', async () => {
+      const invalidAddress = FAKE_ADDRESS;
+
+      await expect(escrowClient.abort(invalidAddress)).rejects.toThrow(
+        ErrorInvalidEscrowAddressProvided
+      );
+    });
+
+    test('should throw an error if an hasEscrow return false', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(false);
+
+      await expect(escrowClient.abort(escrowAddress)).rejects.toThrow(
+        ErrorEscrowAddressIsNotProvidedByFactory
+      );
+    });
+
+    test('should successfully abort escrow', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await escrowClient.abort(escrowAddress);
+
+      expect(escrowClient.escrowContract.abort).toHaveBeenCalledWith();
+    });
+
+    test('should throw an error if the abort fails', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+      escrowClient.escrowContract.abort.mockRejectedValueOnce(new Error());
+
+      await expect(escrowClient.abort(escrowAddress)).rejects.toThrow();
+
+      expect(escrowClient.escrowContract.abort).toHaveBeenCalledWith();
+    });
+  });
+
+  describe('addTrustedHandlers', () => {
+    test('should throw an error if escrowAddress is an invalid address', async () => {
+      const escrowAddress = FAKE_ADDRESS;
+      const trustedHandlers = [ethers.constants.AddressZero];
+
+      await expect(
+        escrowClient.addTrustedHandlers(escrowAddress, trustedHandlers)
+      ).rejects.toThrow(ErrorInvalidEscrowAddressProvided);
+    });
+
+    test('should throw an error if an hasEscrow return false', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const trustedHandlers = [ethers.constants.AddressZero];
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(false);
+
+      await expect(
+        escrowClient.addTrustedHandlers(escrowAddress, trustedHandlers)
+      ).rejects.toThrow(ErrorEscrowAddressIsNotProvidedByFactory);
+    });
+
+    test('should throw an error if an trusted handlers length to equal 0', async () => {
+      const escrowAddress = ethers.constants.AddressZero;
+      const trustedHandlers = [];
+
+      escrowClient.escrowFactoryContract.hasEscrow.mockReturnValue(true);
+
+      await expect(
+        escrowClient.addTrustedHandlers(escrowAddress, trustedHandlers)
+      ).rejects.toThrow(ErrorListOfHandlersCannotBeEmpty);
+    });
+
+    test('should throw an error if the trusted handlers contain invalid addresses', async () => {
```

Comment:
fix writing:
'should throw an error if trusted handlers contains invalid addresses'